.github/workflows/workflow.yml

```diff
@@ -26,6 +28,7 @@ jobs:
 
   release:
     if: github.ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
```

Comment:
Similar: GitHub UI pointed out that this was missing, so I thought I'll fix it here.